Dockerfile

```diff
@@ -12,47 +12,53 @@ ENV PATH /tmp/google-cloud-sdk/bin:$PATH
 
 WORKDIR /tmp
 
-# Buildpack-deps basic requirements
+# Install dependencies
 RUN apk add --update --no-cache \
         ca-certificates \
         curl \
+        python \
         gnupg \
         wget \
         tar \
-        unzip && \
-        apk add --update --no-cache \
+        unzip \
+        py-crcmod \
+        libc6-compat \
+        openssh-client \
         git \
-        bash && \
-        curl --location https://download.docker.com/linux/static/edge/x86_64/docker-$DOCKER_CLIENT_VERSION.tgz --output /tmp/docker-$DOCKER_CLIENT_VERSION.tgz && \
+        bash
```

Comment:
I would order these alphabetically for easier further maintenance

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments

Reply:
Hope i did it well :D